### PR TITLE
add short reference list about handling PRs

### PR DESCRIPTION
**Pull request by DivineDominion**
After my personal fiasco in #262, here's a proposal to remind folks like me how to properly maintain this repo :)

Anything left unsaid? Other ideas or suggestions?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -120,3 +120,13 @@ cd ReSwift-gh-pages
 # Documentation is ready to be committed.
 ```
 
+## Pull Request Technicalities
+
+Here are some hard facts for reference:
```
Comment (Ben-G) [inline CONTRIBUTING.md]: I would rephrase this to something that sounds a little friendlier :) Or maybe just remove this sentence and jump right into the bullet point list.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -120,3 +120,13 @@ cd ReSwift-gh-pages
...
+
+* You're always welcome to open Pull Requests, even if the result is not quite finished or you need feedback!
+* Run Swift Lint locally to ensure there are no code-level oddities.
```
Comment (Ben-G) [inline CONTRIBUTING.md]: nit `SwiftLint`

Comment (Ben-G) [inline CONTRIBUTING.md]: Also, I would rephrase to something like this:

`Run SwiftLint locally to ensure your code-style is consistent with the rest of the codebase`.

**Comment by DivineDominion** [comment]
Thanks for the feedback, @Ben-G! I did just rephrase it a bit.
